Rules/CommonScripts/accolade_data.cfg

```
@@ -83,10 +107,10 @@ Blue_Tiger =
 	# participation 4 - EU 2v2 archer tournament no. 2
 
 Bunnie =
-	customhead TournamentHeads 1 1548201600 12;
-	gold 5;
+	customhead TournamentHeads 1 1548201600 15;
+	gold 6;
 	silver 5;
-	participation 17;
+	participation 18;
```

```suggestion
	participation 19;
```
You miscounted here.